data/tlg0645/tlg003/__cts__.xml

```diff
@@ -1,7 +1,7 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0645" xml:lang="grc" urn="urn:cts:greekLit:tlg0645.tlg003">
   <ti:title xml:lang="eng">Dialogue with Trypho</ti:title>
   <ti:edition urn="urn:cts:greekLit:tlg0645.tlg003.perseus-grc2" workUrn="urn:cts:greekLit:tlg0645.tlg003">
-    <ti:label xml:lang="fre">Dialogue avec Tryphon</ti:label>
-    <ti:description xml:lang="fre">Justin, Dialogue avec Tryphon, Georges Archambault, Picard, 1909</ti:description>
+    <ti:label xml:lang=";at">Dialogus cum Tryphone</ti:label>
```

Comment:
the failing error is here

Reply:
Oops, how did I miss that. Thanks so much for catching.